docs/operator-guide/cd/transfer-variables.md

```diff
@@ -0,0 +1,33 @@
+# Transfer Variables Into Deploy Pipelines
+
+KubeRocketCI allows to inject variables into deploy pipelines via KubeRocketCI portal. With this feature, users can specify key-value pairs that will be transferred into the deploy pipeline as environment variables. This feature is especially helpful when pipelines have specific variables that must be unique for each new pipeline run. In this case, users will need to simply specify the variables in UI, instead of creating new cumbersome pipelines with new variables.
```

Comment:
lets avoid to use 'cumbersome'

Reply:
please remove
"""
This feature is especially helpful when pipelines have specific variables that must be unique for each new pipeline run. In this case, users will need to simply specify the variables in UI, instead of creating new cumbersome pipelines with new variables.
"""